content/national-qualifications-framework/ncit/content/ethics-and-professionalism/_index.md

```diff
@@ -84,7 +84,7 @@ ISACA works in the areas of Information Technology (IT), IT Risk, IT Governance
 
 **Interactive Entertainment South Africa (IESA)**
 
-http://www.iesa.org.za/
+https://web.archive.org/web/20171115033111/http://www.iesa.org.za/
 
 IESA supports interactive entertainment companies and affiliate organisations through research grants, bursary programmes, trade missions and lobbying of government for better legislation and support around video games.
```

Comment:
@ry-oc @JacobUmuzi Not sure if this organisation exists anymore? We have two options here, just link to their most recent captured working version of site (which is what I've done here) or find out if they still exist and have a different name or site; if not, just remove this section from the material.

Reply:
@ry-oc I would suggest removing the part, I am almost certain it seized to exist.

Reply:
I think we can just remove this.